src/de.rs

```diff
@@ -51,6 +51,15 @@ impl KeyDeserialize for &[u8] {
     }
 }
 
+impl<const N: usize> KeyDeserialize for [u8; N] {
+    type Output = Vec<u8>;
+
+    #[inline(always)]
+    fn from_vec(value: Vec<u8>) -> StdResult<Self::Output> {
+        Ok(value)
+    }
+}
```

Comment:
Is `Output = Vec<u8>` really what we want here? Is it possible to use `[u8; N]` and `&[u8; N]` as a map key with this? Somewhere needs to be the code that deserialized into arrays when looing through map entries, right?

Reply:
Good point. It is possible to use `[u8; N]` and `&[u8; N]` as map keys with this version (for that only the `PrimaryKey` trait is needed), but I think you are right, `Output = [u8; N]` is the correct thing to use here.
I also just noticed that I should probably implement `KeyDeserialize` for `&[u8; N]`, otherwise it's not possible to range through a `Map<&[u8; N], _>`.